routes/web.php

```diff
@@ -393,6 +393,13 @@ function (): void {
                             ->where('jobPoster', '[0-9]+')
                             ->middleware('can:delete,jobPoster')
                             ->name('manager.jobs.destroy');
+                        Route::post(
+                            'jobs/{jobPoster}/set-status/{status}',
```

Comment:
Good point. Renamed the routes to `jobs/{jobPoster}/status/{status}`. Switched the api route to a PUT, since that seems more appropriate, but left the HR and Manager portal POST routes for form requests. Also made sure that the controller can gracefully handle a request to set the status to what the job already has.